server/backend/api/app/routers/router_guards.py

```diff
@@ -114,6 +174,7 @@ def authorize_by_application_role(
     role: FamRole = Depends(get_request_role_from_id),
     db: Session = Depends(database.get_db),
     claims: dict = Depends(validate_token),
+    requester: Requester = Depends(get_current_requester),
```

Comment:
we need to add requester Depends in this authorize_by_application_role method, because when we call authorize_by_app_id inside this method, the default Depends for authorize_by_app_id will not work automatically, we need to pass the requester as a parameter